crates/bevy_mikktspace/src/lib.rs

```diff
@@ -4,8 +4,7 @@
     clippy::undocumented_unsafe_blocks,
     clippy::ptr_cast_constness
 )]
-// FIXME(3492): remove once docs are ready
-#![allow(missing_docs)]
+#![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]
```

Comment:
This is mega-pedantic so feel free to ignore:
This reason message has a grammatical error called a comma splice, where two independent clauses are joined by a comma.
2 possible fixes:
 - "Not all docs are written yet. See #3492."
 - "Not all docs are written yet; see #3492."

Reply:
I take your point, but given it doesn't interfere with comprehension in this case I'll turn a blind eye to grammar :slightly_smiling_face: